content/v2.0/reference/flux/stdlib/built-in/transformations/type-conversions/duration.md

```diff
@@ -24,6 +24,12 @@ duration(v: "1m")
 ### v
 The value to convert.
 
+_**Data type:** Integer | String | Uinteger_
+
+{{% note %}}
+`duration()` assumes numeric input values are **nanoseconds**.
```

Comment:
this is just a strange function.   I looked at the code.  Numeric input is presumed to be a quantity of `nanoseconds` like you said.   String input would be parsed in the form of  `3d6h5m2s` as specified here:  https://github.com/influxdata/flux/blob/master/docs/SPEC.md#duration-literals